plugins/modules/monitoring/newrelic_deployment.py

```diff
@@ -46,7 +46,7 @@
     type: str
     description:
       - A revision number (e.g., git commit SHA)
-    required: false
+    required: True
```

Comment:
Always use `true` and `false` for YAML booleans:
```suggestion
    required: true
```